src/_posts/databases/influxdb/2000-01-01-dump-restore.md

```diff
@@ -29,10 +29,16 @@ Daily backups done by Scalingo are listed in the database specific dashboard:
 
 ### Restore
 
+Note: if you need to restore a backup on your Scalingo server, please contact our
```

Comment:
suggestion(non-blocking): I would be more specific. "server" seems a bit too vague.

```suggestion
Note: if you need to restore a backup on your Scalingo for InfluxDB®, please contact our
```